src/core/core.controller.js

```diff
@@ -101,6 +101,20 @@ function determineLastEvent(e, lastEvent, inChartArea, isClick) {
   return e;
 }
 
+function getDatasetArea(meta) {
+  const {iScale, vScale} = meta;
```

Comment:
could just use xScale/yScale to avoid the horizontal checks?

Reply:
@kurkle sorry, I haven't seen your review. It's not clear to me why we should avoid horizontal checks. As far as I know, scaled scales could be horizontal too.

Reply:
Now I have understood what you meant.... :( I'm doing

Reply:
xScale is horizontal, yScale is vertical. just for less lines / complexity.

Reply:
done